cmd/security-agent/api/agent/agent.go

```diff
@@ -125,7 +125,7 @@ func (a *Agent) makeFlare(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	logFile := config.Datadog.GetString("log_file")
```

Comment:
We should have a different config setting (maybe `security_agent.log_file`), otherwise the Agent and Security Agent will log to the same file if `log_file` is set.

Reply:
done